src/extension/gogoduck/src/main/java/au/org/emii/gogoduck/worker/GoGoDuckModule_cars.java

```diff
@@ -0,0 +1,12 @@
+package au.org.emii.gogoduck.worker;
+
+public class GoGoDuckModule_cars extends GoGoDuckModule {
```

Comment:
Is this unconventional naming scheme to do with finding the correct module for a given layer?  I can live with it if so.